sql/src/main/java/org/apache/druid/sql/SqlLifecycle.java

```diff
@@ -122,9 +122,7 @@ public String initialize(String sql, Map<String, Object> queryContext)
     if (queryContext != null) {
       newContext.putAll(queryContext);
     }
-    if (!newContext.containsKey(PlannerContext.CTX_SQL_QUERY_ID)) {
-      newContext.put(PlannerContext.CTX_SQL_QUERY_ID, UUID.randomUUID().toString());
-    }
+    newContext.putIfAbsent(PlannerContext.CTX_SQL_QUERY_ID, UUID.randomUUID().toString());
```

Comment:
Another good candidate for `computeIfAbsent`.

Reply:
Done.